app/views/unattended/provisioning_templates/snippet/ca_registration.erb

```diff
@@ -16,16 +16,21 @@ description: |
   if [ -f "$KATELLO_SERVER_CA_CERT" ]; then
     <%= save_to_file('"$KATELLO_SERVER_CA_CERT"', foreman_server_ca_cert) -%>
 
-    if [ -f /etc/debian_version ]; then
-      CA_TRUST_ANCHORS=/usr/local/share/ca-certificates/
+    . /etc/os-release
+
+    if [ "$ID" = "debian" ] || [ "$ID_LIKE" = "debian" ]; then
+      # update-ca-certificates is picky and only looks at .crt files
+      CA_TRUST_ANCHOR_CERT=/usr/local/share/ca-certificates/katello-server-ca.crt
```

Comment:
Is this really /use/local? Does this dir always exist or is it necessary to create it?